site/en/blog/chromium-chronicle-19/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "The Chromium Chronicle #19: Faster Development with CLion"
+description: >
+  CLion IDE makes it possible to navigate across symbols, search the codebase,
+  and more.
+layout: 'layouts/blog-post.njk'
+date: 2021-03-30
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/hgu6uTktp2ipmuODZZhP.jpg'
+alt: >
+  Chromium Chronicle image
+tags:
+  - chromium-chronicle
+---
+
+**Episode 19:** by Tom Hughes in Mountain View, CA (March, 2021)<br>
+[Previous episodes](/tags/chromium-chronicle/)
+
+When dealing with large codebases, it's useful to have the code indexed, so you can navigate across symbols. For Googlers, CodeSearch works for Google3 and Chromium, but doesn't yet support Chrome OS, and isn't available to non-Googlers. Don't despair, you can use the CLion IDE (**TODO** go/clion-for-chromeos **TODO**) to get this functionality, in addition to many more useful features.
```

Comment:
I just checked go/clion-for-chromeos  and it's live so may want to remove the TODO

Reply:
It is live, but it's internal only, and this is targeted at non-Google Chrome engineers.